components/brave_extension/extension/brave_extension/elementPicker.html

```diff
@@ -1,130 +1,109 @@
-<head>
-  <meta charset="utf-8">
-  <title>Brave Element Picker</title>
-  <style>
-    :root {
-      color-scheme: light dark;
-    }
+<style>
+  :host {
+    background: transparent;
+    color: black;
+    font: 14px sans-serif;
+    height: 100%;
+    width: 100%;
+    margin: 0;
+    overflow: hidden;
+    font-family: "Noto Sans", Arial, sans-serif;
+  }
 
-    body {
-      background: transparent;
-      color: black;
-      font: 14px sans-serif;
-      height: 100%;
-      width: 100%;
-      margin: 0;
-      overflow: hidden;
-      font-family: "Noto Sans", Arial, sans-serif;
-    }
+  :host :focus {
+    outline: none;
+  }
 
-    body :focus {
-      outline: none;
-    }
+  #rules-box {
+    height: 8em;
+    padding: 5px;
+    margin-bottom: 5px;
+    background-color: #fff;
+    border-radius: .1em;
+    overflow-y: auto;
+    overflow-wrap: break-word;
+  }
 
-    #rules-box {
-      height: 8em;
-      padding: 5px;
-      margin-bottom: 5px;
-      background-color: #fff;
-      border-radius: .1em;
-      overflow-y: auto;
-      overflow-wrap: break-word;
-    }
+  #rules-box>textarea {
+    height: 100%;
+    width: 100%;
+    resize: none;
+    border: none;
+    overflow: auto;
+  }
```

Comment:
To fix the double parent + child scroll bars, remove padding and both overflow properties from this rule and add them to the textarea rule below. In addition, add box-sizing: border-box; to the textarea rule.

![image](https://github.com/user-attachments/assets/95119e3a-3988-43d4-b470-ee47479ced2f)


```
  #rules-box {
    height: 8em;
    margin-bottom: 5px;
    background-color: #fff;
    border-radius: .1em;
  }

  #rules-box>textarea {
    box-sizing: border-box;
    height: 100%;
    width: 100%;
    resize: none;
    border: none;
    padding: 5px;
    overflow: auto;
    overflow-wrap: break-word;
  }
```

<img width="394" alt="image" src="https://github.com/user-attachments/assets/dd6290dc-196e-42c6-94a0-3d2e11470a25">

Reply:
done.
thanks for the suggestion.